app/src/main/java/org/sopt/alami/presentation/setting/SettingScreen.kt

```diff
@@ -86,24 +156,18 @@ fun SettingScreen(
         }
         item {
             Spacer(modifier = Modifier.height(40.dp))
-            AlarmPicker()
+            AlarmPicker(viewModel = viewModel)
         }
+
         item {
             Spacer(modifier = Modifier.height(40.dp))
             ScreenDivider()
         }
-        item {
-            DateSelect()
-        }
-        item {
-            ScreenDivider()
-        }
-        item {
-            AddMission()
-        }
-        item {
-            ScreenDivider()
-        }
+        item { DateSelect() }
+        item { ScreenDivider() }
+        item { AddMission() }
+        item { ScreenDivider() }
```

Comment:
p2
```suggestion
        item {
            DateSelect()
            ScreenDivider()
            AddMission()
            ScreenDivider()
            SoundProgress(currentPosition = 1f, onSeek = {})
        }
```

이렇게 하는건 어떨까요?